src/components/Integrations/Shopify/Shopify.js

```diff
@@ -30,29 +30,120 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
     getData();
   }, [shopifyClient]);
 
+  const shopifyHeader = (
+    <>
+      <div className="block">
+        <h2>Panel de Control - Shopify</h2>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequuntur facere velit
+          officia quibusdam perspiciatis impedit veritatis et, ducimus dolorem{' '}
+          <a href="http://fromdoppler.com">corporis</a>.
```

Comment:
dont need link in english too?

Reply:
I have not language yet. This is only visual styles.